changelogs/unreleased/5586-deveshk0-minor.md

```diff
@@ -0,0 +1,27 @@
+## HTTPProxy: Allow custom host header on HttpProxy level.
+
+This Change allows you set custom host headers on httpProxy level, Please note headers are appended to requests/responses in the following order: weighted cluster level headers, route level headers, virtual host level headers and finally global level headers.
+
+#### Example
+```yaml
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
+metadata:
+  name: custom-host-header
+spec:
+  fqdn: local.projectcontour.io
+   requestHeadersPolicy:
```

Comment:
I think there is an indentation issue here in the example.

Reply:
Fixed